MovieKuno/Tab Pages/Home Views/HomeView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  HomeView.swift
+//  MovieKuno
+//
+//  Created by Leah Joy Ylaya on 1/20/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct HomeView: View {
+    @State private var movies: Movies = [
+        Movie(id: 1,
```

Comment:
To stress @AJCordova's suggestion, you can create a static variable in the model to create a template for this